.gitattributes

```diff
@@ -12,3 +12,4 @@
 *.ttf binary
 *.woff binary
 *.woff2 binary
+*.tgz binary
```

Comment:
This can be useful in the future (although not needed for now). considering `tgz` as anything else than binary will most likely break the archive